src/commands/dsyms/upload.ts

```diff
@@ -89,14 +89,18 @@ export class UploadCommand extends Command {
     this.config = await resolveConfigFromFileAndEnvironment(
       this.config,
       {
-        apiKey: process.env.DATADOG_API_KEY,
-        datadogSite: process.env.DATADOG_SITE,
+        apiKey: process.env.DATADOG_API_KEY || process.env.DD_API_KEY,
```

Comment:
`this.config.apiKey` is used at line 115 in `newApiKeyValidator()`. Can you update the related error message in `src/helpers/apikey.ts` at line 54?

I suggest:
```js
throw new InvalidConfigurationError(
  `Neither ${chalk.red.bold('DATADOG_API_KEY')} nor ${chalk.red.bold('DD_API_KEY')} contains a valid API key for Datadog site ${this.datadogSite}`
)
```